Makefile

```diff
@@ -0,0 +1,300 @@
+endef
+
+define print_list_of_connectors
+	echo "   $1"
+endef
+
+help: .pw_logo
+	@echo
+	@echo "  Targets"
+	@echo "  ======="
+	@echo "     pipelinewise                                               Install the main PipelineWise component"
+	@echo "     pipelinewise_no_test_extras                                Install the main Pipelinewise component without test extras"
+	@echo
+	@echo "     all_connectors                                             Install all connectors"
+	@echo "     extra_connectors"                                          Install only extra connectors
+	@echo "     connectors -e pw_connector=connector1,connector2,...       Install specific connector(s)"
+	@echo
+	@echo "     list_components                                            Show a list of installed components"
+	@echo "     list_default_connectors                                    Show a list of available default connectors"
+	@echo "     list_extra_connectors                                      Show a list of available extra connectors"
+	@echo
+	@echo "     clean_all                                                  Clean all installed components"
+	@echo "     clean -e pw_connector=connector1,connector2,...            Clean a specific connector(s)"
+	@echo
+	@echo "   Options"
+	@echo "   ======="
+	@echo "      -e pw_connector=connector1,connector2,...                 Define a list of connectors for installing or cleaning"
+	@echo "      -e pw_acceptlicenses=y/Y/Yes/YES                           Forcing to accept the licenses automatically"
+	@echo
+	@echo "   To start CLI"
+	@echo "   ============"
+	@echo "      $$ source $(VENV_DIR)/pipelinewise/bin/activate"
+	@echo "      $$ export PIPELINEWISE_HOME=$(PIPELINEWISE_HOME)"
+	@echo "      $$ pipelinewise status"
+	@echo
+	@echo "--------------------------------------------------------------------------"
+
+
+pipelinewise: .check_gettext .pw_logo
+	$(call make_virtualenv,pipelinewise)
+	$(call print_execute_time,PipelineWise)
+
+pipelinewise_no_test_extras: .set_pip_args pipelinewise
+
+clean_all:
+	@echo -n "Cleaning previous installations in $(VENV_DIR)..."
+	@rm -rf $(VENV_DIR)
+	@echo -e "$(OK_MSG)"
+
+clean:
+ifeq ($(pw_connector),)
+	@echo "use -e pw_connector=connector1,connector2,...."
+	@exit 1
+endif
+	$(eval space:= )
+	$(eval space+= )
+	$(eval comma:=,)
+	$(eval connectors_list:=$(subst $(comma),$(space),$(pw_connector)))
+
+	@$(foreach var,$(connectors_list), $(call clean_connectors,$(var));)
+
+connectors: .check_license_env_var
+ifeq ($(pw_connector),)
+	@echo "use -e pw_connector=connector1,connector2,...."
+	@exit 1
+endif
+	$(eval space:= )
+	$(eval space+= )
+	$(eval comma:=,)
+	$(eval connectors_list:=$(subst $(comma),$(space),$(pw_connector)))
+
+	@$(foreach var,$(connectors_list), $(call install_connectors,$(var));)
+	$(call print_execute_time,Connectors)
+
+
+all_connectors: .check_license_env_var default_connectors extra_connectors
```

Comment:
is calling `.check_license_env_var` necessary here? it's redundant given that  both `default_connectors` and `extra_connectors` are doing it.

Reply:
It's also failing when I run it: 
```
$ make all_connectors
Makefile:240: *** missing separator.  Stop.
```

Reply:
All fails caused by that wrong placed double quotes. should be solved now. (very strange it was not failing on Mac and CI!)